uktrade/data-hub-frontend
# Add Export Win - Officer details (1 of 6 steps)

## Description of change
This is step 1 of 6 of the Export Win user journey that enables Lead Officers to add an Export Win to Data Hub. 

## Test instructions

Go to `/exportwins/create?step=officer_details&company=<company-uuid>`

1. Enter all 4 fields (the last field is optional)
2. Click continue
3. Click back
4. Ensure all data entered from 1. has persisted.

**HQ team, region or post only appears once the user has selected a Team Type.** 

## Screenshots
<img width="988" alt="officer-details" src="https://github.com/uktrade/data-hub-frontend/assets/964268/574d7fc4-648e-4f73-9c53-c9bb220579de">
<img width="997" alt="validation" src="https://github.com/uktrade/data-hub-frontend/assets/964268/fa0dc3b3-1cd3-4754-90a7-7d0c788d6ee0">

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
